setup.cfg

```diff
@@ -156,6 +156,9 @@ install_requires =
     typing-extensions>=3.7.4;python_version<"3.8"
     unicodecsv>=0.14.1
     werkzeug~=1.0, >=1.0.1
+    # needed for generating virtual python environments when running tasks
+    virtualenv>=20.4.3
+    psycopg2>=2.8.6
```

Comment:
Why is psycopg needed?

Reply:
When testing we had an exception due to the module missing. Maybe there is a dependency that only needs it on Windows. Should I remove it and just manually install it if needed or is there an option to only install it on Windows?

Reply:
It depends on why th module is needed. But I’m quite sure adding this as a harde dependency to Airflow core is the wrong solution.

Reply:
They should both be removed from here. They are already  present as extras. You should simply install airflow with postgres and virtualenv extras:

```
pip install "apache-airflow[postgres,virtualenv]" --constraint \
https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt
```